pkg/oc/cli/process/process.go

```diff
@@ -303,38 +374,51 @@ func RunProcess(f kcmdutil.Factory, in io.Reader, out, errout io.Writer, cmd *co
 		}
 	}
 
-	outputFormat := kcmdutil.GetFlagString(cmd, "output")
-	if outputFormat == "describe" {
+	if o.outputFormat == "describe" {
```

Comment:
after we break the back of this, we should sweep and start using the output held by the flags themselves.